### ISSUE TITLE
fid problem

### ISSUE THREAD

**Issue by deadsmither5** [puar-playground/LACE#1]
hi,sorry to bother you, i'm runing your code now,but when I use the checkpoints for rico25, and run the test commandline,the fid score is about 13 which larger than it in your paper，so I'm wondering  whether the checkpoints you provided is the best version or not. hope you can answer my question,thanks.

**Comment by puar-playground** [comment]
Hi,

I have reproduced the error you encountered. Thanks for your interests and pointing out the error.

It is due to the use of overlap constraint in the post processing function. In fact, If you remove the "--beautify" flag, you will see the fid is as reported. We have modified the "test.py" script to automatically remove the overlap constraint during post-processing of rico25 checkpoints (see line 73, 159). Now the test code should output the correct results.

Since overlapping is prevalent in the Rico dataset. We do not use the overlap constraint in training or post-processing on Rico dataset.
